modules/47_regipol/module.gms

```diff
@@ -8,11 +8,26 @@
 
 *' @title Regional Policies
 *'
-*' @description  The 47_regipol module includes region specific policies.
+*' @description  The 47_regipol module includes region-specific policies and adjustments.
 *'
 *'
-*'               The `regiCarbonPrice` realization allows to determine region specific year or budget targets for CO2 or GHG emissions.
-*'               It overwrites all other carbon prices for selected regions (for example those implemented by 45_carbonprice or 46_carbonpriceRegi).
+*'               The `regiCarbonPrice` realization has two purposes. First, it allows to determine region specific year or budget targets for CO2 or GHG emissions.
+*'               Second, it comprises region-specific adjustments that are always active in this realization and policies that can be activated by specific switches (see bounds file).
+*'               On the emissions targets: Emissions targets can be activated via the switch cm_emiMktTarget and there are number of options that can be chosen to specify them
+*'               like the type of target (annual target or budget), the target year, the regions the target should be applied to,
+*'               the emissions metric (CO2, GHG, CO2 excl. bunkers/LULUCF etc.) and the emissions market (all, ETS, ESR). For details on the options please see the description of the switch.
+*'               When an emissions target is set via this switch, the carbon price in the target years and regions, to which it is applied, is adjusted over nash iterations in REMIND
+*'               until the desired emissions target is reached within a certain margin of tolerance defined by cm_emiMktTarget_tolerance. Note that also multiple emissions targets can be specified for
+*'               different years (e.g. the EU's 2030 and 2050 targets). The carbon price trajectory up to the target years are linear and the carbon price
```

Comment:
great so far! I remember reading a short example in main.gms using EUR multiple targets, could we put that here as well? maybe use a somewhat complicated example like cumulative emission target before 2030, and net-zero by 2050, or something like that, so users intuitively understands the full flexibility of this switch